dataswarm/manager/dataswarm_file.h

```diff
@@ -0,0 +1,10 @@
+#ifndef DATASWARM_FILE_H
```

Comment:
For consistency with other modules, this should be `dataswarm_file_declare`, `dataswarm_file_commit` etc